python/tvm/relay/build_module.py

```diff
@@ -296,6 +299,28 @@ def optimize(mod, target=None, params=None):
     return mod, params
 
 
+def bind_params_by_name(func, params):
```

Comment:
Could we move this change to expose bind_params_by_name into its own PR? I could also make use of this more generally so I don't think it's specific to this test case.

Reply:
ok. I need to think about how to test this function.